tests/generic_config_updater/test_aaa.py

```diff
@@ -0,0 +1,609 @@
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success, expect_op_failure
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
+pytestmark = [
+    pytest.mark.topology('t0'),
```

Comment:
>t0 [](http://example.com/codeflow?start=26&length=2)

change it to `any`?

Reply:
Not sure if other topos have the same init config with t0. 
For example, t0 dont have AAA or TACPLUS table in configDB, so to make GCU change on that, some test need to add init config to initialize the table

Reply:
Any topo should have the same init AAA config. Please let me know it is not.

Reply:
Apply AAA test to topo `any`